src/components/import/render-utils.tsx

```diff
@@ -35,3 +37,31 @@ export const routeDecoratorIcon = (
       return <GitAltIcon style={{ fontSize: radius }} title={t('devconsole~Edit source code')} />;
   }
 };
+
+export type BreadCrumbsProps = {
+  breadcrumbs: { name: string; path: string }[];
+};
+
+export const BreadCrumbs: React.SFC<BreadCrumbsProps> = ({ breadcrumbs }) => (
```

Comment:
This is not in the SDK?

Reply:
Not really. And the kubevirt plugin is not using it either.

Reply:
Could you check if you can use Breadcrumb from patternfly ?  I think the kubevirt plugin is using it.

Reply:
Thanks for the advice. Updated the reference and the component.

Reply:
Looks good.